Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
```

Comment:
Good consistency with rest of the code, and it is a nice addition to the program.